webpack.commons.js

```diff
@@ -58,36 +58,40 @@ module.exports = {
     }),
   ],
   devServer: {
-    contentBase: path.join(__dirname, 'dist'),
-    contentBasePublicPath: BASE_HREF,
+    contentBase: [path.join(__dirname, 'dist'), path.resolve(__dirname, 'node_modules', 'esn-frontend-login', 'dist')],
```

Comment:
IQ level infinity. I never thought such a thing was possible. But this is just a temporary solution to avoid the hassle of copy-pasting cookies, right?